IMC_R/EDA_Raymond/EDA-R.md

```diff
@@ -0,0 +1,123 @@
+# Exploring Movie Streaming Data With R
+Raymond Wang
+
+## Exploratory Data Analysis
+Exploratory data analysis, or EDA for short, is an approach to exploring data that can help you obtain basic but essential insights before performing more complex operations. Although the techniques employed by EDA are relatively simple, EDA is a critical component of any statistical analysis. For starters, it allows us to understand the underlying structure of the data and evaluate the validity of our assumptions. It can also help us formulate hypotheses about the data, which we can then test with more defined models. In this blog, I will show a few examples of EDA using R, the programming language of choice for many data scientists. If you’d like to try out these examples for yourself, make sure you have R installed (see [here](https://www.r-project.org/) for details on how to do this). When actually coding, I like using Jupyter Notebook, but you can use any environment you’d like (RStudio is another great option).
+
+## EDA Techniques
+If you’ve ever worked with data before, you’ve likely used some EDA techniques, even if you’ve never heard of the term. That’s because EDA is an incredibly broad topic, and there’s usually no single model or method that defines your approach. Many common techniques are graphical in nature; histograms, box plots, and scatter plots all help us understand data visually. Tabular results describing correlations, averages, and other summary statistics are also incredibly helpful. For this blog, I will be using a movie streaming dataset, courtesy of Kaggle, which you can download for yourself [here](https://www.kaggle.com/ruchi798/movies-on-netflix-prime-video-hulu-and-disney). I’ll also be using some packages from Tidyverse to help with the analysis.
+
```

Comment:
I think some introduction of EDA and tidyverse can be rearranged here. "if you've ever worked with data..." can probably appear earlier. "Traditional approaches to exploring data that can help you obtain basic but essential insights such as finding summary statistics are all techniques that define EDA...." Gradually introduce Tidyverse before introducing dataset. "One such package that can enable all of the said techniques is Tidyverse...."